tools/changelog/0.9.6.md

```diff
@@ -0,0 +1,26 @@
+# FORGE.js changelog 0.9.6
+
+### Viewer
+
+### Media
+
+### Renderer
+
+### Display
+
+### Camera
+
+### Views
+
+### Video
+
+### Hotspots
+
+### Plugins
+
+### Samples
+
+### Misc
+
+### Diff since 0.9.4
```

Comment:
You've missed this `0.9.4`

Reply:
Yeah saw that after merge. Not a big deal.
Hope you're doing well dude ;)